test/ctia/test_helpers/core.clj

```diff
@@ -279,10 +279,11 @@
   to make them explicit."
   ([t-with-app :- (s/=> s/Any
                         (s/named s/Any 'app))]
-   (fixture-ctia-with-app t-with-app true))
+   (fixture-ctia-with-app t-with-app true {}))
   ([t-with-app :- (s/=> s/Any
                         (s/named s/Any 'app))
-    enable-http?]
+    enable-http? :- s/Bool
+    services-override :- (s/maybe {s/Keyword s/Any})]
```

Comment:
Nice, I think this is the first time the `{Kw Service}` representation comes in handy in CTIA!

I suggest:
1. keep the 2-argument arity (make backwards compatible).
2. (Later) Creating a new function that switches the arguments to your liking.
3. (Later) Gradually moving to that function.

Reply:
> keep the 2-argument arity (make backwards compatible).
> (Later) Creating a new function/macro that switches the arguments to your liking.

Initially, I thought about doing just that, but then I looked through every invocation, and it 2-arity version used only in a couple of places (already fixed in this PR).

I'm not very fond of this function because it acts like a macro, albeit a weird one, where the body isn't in the tail position. I'll make a PR deprecating it with a new version to use instead.

Reply:
Are you suggesting to limit the schema to contain only service keys, like here https://github.com/advthreat/iroh/blob/master/services/status/src/iroh/status/schemas.clj#L26

Reply:
> Are you suggesting to limit the schema to contain only service keys?

No, it's fine. I just came up with the service map representation as an _input_ in the context of CTIA, but we've only ever used it in user code in our other project until now.